pkg/pod/entrypoint.go

```diff
@@ -19,48 +35,46 @@ const (
 	downwardVolumeName     = "downward"
 	downwardMountPoint     = "/builder/downward"
 	downwardMountReadyFile = "ready"
-	ReadyAnnotation        = "tekton.dev/ready"
-	ReadyAnnotationValue   = "READY"
+	readyAnnotation        = "tekton.dev/ready"
```

Comment:
I wonder if we shouldn't keep this exported (mainly as a "documentation" and could be use by dependencies ? (a bit like `ManagedByLabelKey`)

Reply:
I'd prefer dependents not to know/rely on any behavior of the Pod we manage, because it makes it much harder to change in the future.

Ideally users wouldn't even have to be aware there's a Pod down there _at all_, let alone how Tekton signals details of that Pod to itself.